### PR TITLE
Switch to static SVG badges

### PR DESCRIPTION
**Pull request by jerivas**
This completely removes the shields.io dependency and simply serves badges as static SVGs. The badges no longer display the version slug, they just show the "unknown", "passing", and "failing" status, with the appropriate color.

Docs have also been updated.

### CODE REVIEW

**Comment by ericholscher** [comment]
Thanks for this @jerivas! If you want to fix the linting issue, I'll get it merged & deployed. I'll give ya a couple days, and then fix it myself if I don't hear from ya :)

File: readthedocs/projects/views/public.py [hunk 1]
```diff
@@ -105,36 +106,23 @@ def get_context_data(self, **kwargs):
 @never_cache
 def project_badge(request, project_slug):
     """Return a sweet badge for the project"""
+    BADGE_PATH = "projects/badges/%s.svg"
```
Comment (ericholscher) [inline readthedocs/projects/views/public.py]: linting says this should be lower case.

Comment (ericholscher) [reply readthedocs/projects/views/public.py]: I also believe this probably needs to have a MEDIA_URL prepended to it, for production.

**Comment by jerivas** [comment]
Just committed the fix 😄

**Comment by ericholscher** [comment]
Thanks! :)

**Comment by wmayner** [comment]
This removes support for the `style` parameter that Shields.io provides. Should we make that available?

**Pull request merged**